pkgs/servers/scaphandre/default.nix

```diff
@@ -13,16 +13,22 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "scaphandre";
-  version = "0.5.0";
+  version = "1.0.0";
 
   src = fetchFromGitHub {
     owner = "hubblo-org";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-cXwgPYTgom4KrL/PH53Fk6ChtALuMYyJ/oTrUKHCrzE=";
+    hash = "sha256-Y3TZJJ6ypZGGIEApDqiID7xWOSFTVjNQeDSomR3v5gY=";
   };
 
-  cargoSha256 = "sha256-Vdkq9ShbHWepvIgHPjhKY+LmhjS+Pl84QelgEpen7Qs=";
+  patches = [ ./cargo-version.patch ];
```

Comment:
Can we put https://github.com/hubblo-org/scaphandre/commit/eebb06b6ea0d5209c34f88d7a7376a3205305675 and https://github.com/hubblo-org/scaphandre/commit/b27925f3d2ef44359a4f132b5bc1b6b5fc37c4f3 in `cargoPatches` so as not to pull a giant Cargo lock file into Nixpkgs?

Reply:
Sure! I didn't this was available!